crates/network/src/ethereum/builder.rs

```diff
@@ -194,6 +198,10 @@ fn build_1559(request: TransactionRequest) -> Result<TxEip1559, TransactionBuild
     })
 }
 
+const fn will_build_2930(request: &TransactionRequest) -> bool {
```

Comment:
feature req:

`fn can_build_*() -> Result<(), TransactionBuilderError>`

Should return a list of missing keys required to build that transaction type